README.md

```diff
@@ -1,12 +1,53 @@
-## How to set navigation
-- Create a directory hierarchy in `src/pages/config.md`
+# Adobe Connect SDK Documentation
 
-## Launching a deploy
-- Go to Actions > Deployment > Run workflow
+Adobe Connect SDK documentation site deployed to EDS.
+The production address is https://developer.adobe.com/adobe-connect-sdk/
 
-## URL
-- developer-stage.adobe.com/{path-prefix}/{relative-path-to-file}
+## Quick Start
 
-## Where to ask for help
+Start development server:
+```bash
+npm run dev
+```
+Runs Express server on port 3003 serving files from `src/pages/`.
 
-The slack channel #adobe-developer-website is our main point of contact for help. Feel free to join the channel and ask any questions.
+## Commands
+
+**Development**
+- `npm run dev` - Start local server
```

Comment:
Should also make a note about having the runtime connector https://github.com/aemsites/devsite-runtime-connector and the adp-devsite https://github.com/AdobeDocs/adp-devsite also needing to be cloned and with their local servers running